com.unity.render-pipelines.core/Runtime/Common/DynamicResolutionHandler.cs

```diff
@@ -25,6 +25,18 @@ public enum DynamicResScalePolicyType
         ReturnsMinMaxLerpFactor
     }
 
+    /// <summary>
+    /// The source slots for dynamic resolution scaler. Defines registers were the scalers assigned are stored. By default the User one is always used
+    /// </summary>
+    public enum DynamicResScalerSlot
+    {
+        /// <summary> Scaler slot set by the function SetDynamicResScaler</summary>
+        User,
+        /// <summary> Scaler slot set by the function SetSystemDynamicResScaler</summary>
+        System,
+        /// <summary> total number of scaler slots </summary>
+        Count
```

Comment:
Don't need this in C#, you can get the count with stuff like

Enum.GetValues(typeof(DynamicResScalerSlot) ).Length;

Reply:
Still valid ^ :)

Reply:
As discussed left it there. If we dont use it then:
1) We could just not use the Count when we instantiate the array, but then we lose the compiler protection, if say we add another member, the array containing the slots will mismatch. Opening up for a crash
2) If we decide to use reflection, the array containing the slots must then be initialized in a static block, I rather avoid it because every time I use a static block I get some issues with IL2CPP (sometimes reflection data isnt ready) and theres some really strange attributes to add in order to ensure correct initialization
3) An alternative is having a Last member, pointing Last to System, and instantiating the array as Last +1. This sounds a bit too much typing too. In which case for me the best option is #1, drop all compiler safety in favor of cleaner API, since this enum is likely to not be touched further.